LayoutTests/fast/images/decode-static-svg-image-resolve.html

```diff
@@ -0,0 +1,24 @@
+<style>
+    canvas {
+        width: 100px;
+        height: 100px;
+        background-color: red;
+    }
+</style>
+<body>
+    <p>This tests resolving the decode() promise when loading and decoding a static svg image succeeds.</p>
+    <canvas></canvas>
+    <script>
+        if (window.testRunner)
+            testRunner.waitUntilDone();
+        var image = new Image;
+        image.src = "resources/green-400x400.svg";
```

Comment:
This test does not test the `SVGImageElement`. `new Image` creates an `HTMLImageElement`. Decoding the `green-400x400.svg` will in fact do not thing. It will just resolve the promise.

I attached a test case to https://bugs.webkit.org/show_bug.cgi?id=279254.

Reply:
@shallawa - thanks! I only changed two bits, removed `paragraph` and made it as inline comment in CSS and change image reference in your test case. Plus accordingly updated test case expectation file to have two green rects.